app/presenters/result_set_presenter.rb

```diff
@@ -49,10 +50,27 @@ def selected_filter_descriptions
     }.reject(&:empty?)
   end
 
+  def is_best_bet?(score_a, score_b)
+    # We found the average score on the top 500 searches and found that 7 was the most suitable number
+    (score_a / score_b) > 7
+  end
+
+  def get_documents_with_top_result(documents = [])
+    with_top_result(documents) if highlight_top_result?(documents)
+    documents
+  end
+
   def documents
     results.each_with_index.map do |result, index|
+      doc = SearchResultPresenter.new(result).to_hash
+      if highlight_top_result?(results) && index === 0 && result.es_score && results[index+1].es_score
```

Comment:
`results` can be read from anywhere in the class, so it doesn't need to be passed in.

Reply:
I think by moving most of the checks to `highlight_top_result?` you could make this block much easier to read.

The `documents` method could just check `index === 0 && highlight_top_result?`

Then `highlight_top_result?` could do all of the work:

```
    whitelist_finder_for_top_result? &&
      results.length >= 2 &&
      is_best_bet? &&
      sort_option.present? &&
      sort_option["key"].eql?("-relevance")
```